src/CategoryDescription.css

```diff
@@ -0,0 +1,63 @@
+:root {
+  --main-green: #076B49;
+  --sec-green: #74CC67;
+  --white: #FFF;
+  --shadow-color: hsl(0deg 0% 0% / 26%);
+}
+
+.category-container {
+  height: 100vh;
+  width: 100vw;
+  max-width: 768px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.category-container h4 {
+  font-size: 1.5rem;
+  padding-top: 5rem;
+}
+
+.category-container .category-icon {
+  font-size: 10rem;
+  color: var(--main-green);
+}
+
+.category-container p {
+  width: 85%;
+  font-size: 1.2rem;
+  text-align: center;
+}
+
+.category-container .category-buttons {
+  max-width: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-between;
+  margin-bottom: 1rem;
+}
+
+.category-container .category-buttons button {
+  width: 85vw;
+  max-width: 653px;
+  height: 6vh;
+  margin: 0.3rem 0;
+  border: none;
+  border-radius: 0.7rem;
+  font-size: 1.2rem;
+}
+
+.category-container .category-buttons button:hover {
```

Comment:
great css, might be little extra code but it is very clear and scoped to this file great work.

Reply:
Thank you. And thanks for reviewing this PR so quickly!